### PR TITLE
Add notification toast

### LINKED ISSUE #23: Transaction: Toast on tx success/failure
From reqs:

> Toast UI: success or failure toast with link to etherscan / bscscan / polygon / etc

### PR DESCRIPTION
**Pull request by alexkeating**
#### Description
- Add toast for transactions and notifications

Closes #23

### CODE REVIEW

File: components/TxForm.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import useNotifications from "../hooks/useNotifications";
```
Comment (mds1) [inline components/TxForm.tsx:1]: let's stay consistent and use the `@` reference
```suggestion
import useNotifications from "@/hooks/useNotifications";
```

File: hooks/useNotifications.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import { getEtherscanUrl } from "../utils/networks";
```
Comment (mds1) [inline hooks/useNotifications.tsx:1]: same as above
```suggestion
import { getEtherscanUrl } from "@/utils/networks";
```

File: pages/_app.tsx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import Header from "../components/Header";
```
Comment (mds1) [inline pages/_app.tsx:1]: Slipped by on a previous PR
```suggestion
import Header from "@/components/Header";
```

**Review by mds1 — COMMENTED** [review]
Deferring to @wildmolasses for react feedback, just some nits from me. Tested it out and looks good

image

File: components/TxForm.tsx (hunk 1 above)
Comment (alexkeating) [reply components/TxForm.tsx:1]: Ah will fix and use in the future

**Comment by alexkeating** [comment]
Bumping is this good to merge?

**Pull request merged**